test/externalTests/solc-js/solc-js.sh

```diff
@@ -23,41 +16,43 @@
 # You should have received a copy of the GNU General Public License
 # along with solidity.  If not, see <http://www.gnu.org/licenses/>
 #
-# (c) 2017 solidity contributors.
+# (c) 2019 solidity contributors.
 #------------------------------------------------------------------------------
+source scripts/common.sh
+source test/externalTests/common.sh
 
-set -e
-
-if [ ! -f "$1" -o -z "$2" ]
-then
-  echo "Usage: $0 <path to soljson.js> <version>"
-  exit 1
-fi
-
+verify_version_input "$1" "$2"
 SOLJSON="$1"
 VERSION="$2"
 
-DIR=$(mktemp -d)
-(
-    echo "Preparing solc-js (master_060)..."
-    git clone --depth 1 --branch master_060 https://github.com/ethereum/solc-js "$DIR"
-    cd "$DIR"
-    # disable "prepublish" script which downloads the latest version
-    # (we will replace it anyway and it is often incorrectly cached
-    # on travis)
-    npm config set script.prepublish ''
```

Comment:
Why don't we have this in the new `setup` function? It saves time.

Reply:
I added it to `setup_solcjs` because the other external tests can also profit from that.